CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 7.19.0 - 2023/02/22
+
+## Enhancements
+ 
+- Add link to BitBar dashboard for each Appium session [470](https://github.com/bugsnag/maze-runner/pull/470)
+- Set Appium capabilities for the BitBar dashboard [471](https://github.com/bugsnag/maze-runner/pull/471)
+- Clean exit for user readable errors [474](https://github.com/bugsnag/maze-runner/pull/474)
+- Allow `--device` to specify a device (rather than device group) on BitBar [475](https://github.com/bugsnag/maze-runner/pull/475)
+
+## Fixes
+
+- Set capabilities needed for iOS devices on BitBar (with Appium 1.22) [466](https://github.com/bugsnag/maze-runner/pull/466)
+- Tidy capabilities for BitBat devices [473](https://github.com/bugsnag/maze-runner/pull/473)
```

Comment:
```suggestion
- Tidy capabilities for BitBar devices [473](https://github.com/bugsnag/maze-runner/pull/473)
```

Reply:
Haha!  Yes, this is a common typo of mine!